File: Kode/challenge-3/challenge-3.py
n = int(input())
nama = []

for i in range(0,n):
    nama.append(input())

for i in range(0,n):
    urutan = 1
    for j in range(0,i):
        if(nama[i] > nama[j]):
            urutan+=1
        
    print(urutan)

File: Kode/challenge-3/challenge-3.java
import java.util.Scanner;

// Ganti nama file ke 'App.java'
public class App {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();

        String[] nama = new String [n];

        for(int i=0; i<n; i++) {
            nama[i] = input.next();
        }

        for(int i=0; i<n; i++) {
            int posisi = 1;
            for(int j=0; j<i; j++) {
                if((nama[i].compareTo(nama[j]) > 0)) {
                    posisi = posisi + 1;
                }
            }

            System.out.println(posisi);
        }
    }
}
